src/zenml/models/artifact_models.py

```diff
@@ -13,9 +13,10 @@
 #  permissions and limitations under the License.
 """Models representing artifacts."""
 
-from typing import Optional
+from typing import Optional, Union
 from uuid import UUID
 
+from fastapi import Query
```

Comment:
I find it a little strange that our generic model classes (which are also used in the SQLZenStore) now depend on some fastapi class. Can you explain why this is necessary?

Reply:
@AlexejPenner